# frame_type: 0x32(50)
# emgstop: 0 -> EMERGENCYSTOP, else: ERROR
int32 emgstop